runtime/jvmti/jvmtiHook.c

```diff
@@ -582,16 +587,17 @@ jvmtiHookVMStarted(J9HookInterface** hook, UDATA eventNum, void* eventData, void
 
 	if (callback != NULL) {
 		J9VMThread *currentThread = data->vmThread;
-		J9JavaVM *vm = currentThread->javaVM;
 		UDATA hadVMAccess;
 		UDATA javaOffloadOldState = 0;
 		BOOLEAN reportEvent = TRUE;
-
-	 	if (J2SE_VERSION(vm) >= J2SE_19) {
-	 		if (j9env->capabilities.can_generate_early_vmstart == 0) {
+#if JAVA_SPEC_VERSION >= 9
+		J9JavaVM *vm = currentThread->javaVM;
+	 	if (J2SE_VERSION(vm) >= J2SE_19) {	
```

Comment:
The condition `J2SE_VERSION(vm) >= J2SE_19` is redundant if `JAVA_SPEC_VERSION >= 9`.